documentation/SRS.md

```diff
@@ -0,0 +1,126 @@
+# Human resource management web application
+
+Application should provide:
+
+- Storing departments and employees in a database;
+- Display a list of departments and the average salary for each department;
+- Display a list of employees with an indication of the employee's salary;
+- Searching employees born on a certain date or in the period between dates;
+- Updating a list of employees (adding, editing, removing);
+- Updating a list of departments (adding, editing, removing).
+
+## Departments
+### Display a list of departments
+
+The mode is designed to view a list of departments, average salary of departments, and also edit the department's info.
+
+![Main page](mockups/main_page.png)
+
+#### Add department
+#### Main scenario:
+
+- User clicks the “ADD NEW DEPT” button in the list of departments view mode;
+- Application displays form to enter department's title;
+- User enters department's title and presses “Save” button;
+- New department record is successfully added, then the list of departments with added record is displaying.
+
+![Add_new_dept](mockups/add_new_dept.png)
+
+#### Cancel operation scenario:
+
+- User clicks the “ADD NEW DEPT” button in the list of departments view mode;
+- Application displays form to enter department's title;
+- User enters department's title and presses “Cancel” button;
+- Data don’t save in database, then user is redirected to the list of departments.  
+
+#### "Delete DEPT" button
+
+"Delete DEPT" button redirects user to main page with the list of departments without deleted department. 
```

Comment:
At least you should add an alert with confirmation and cancel button. Moreover in case when deletion of a department leads to deletion of all assigned employees - this should be presented in the same alert. Another variant is to add a choice to user:
1. Delete department and make all employees' department equal to null (there is a chance when user would like to delete department and then assign to all employees new department)
2. Delete department with all employees (with confirmation)

And generally, all deletion operations should be followed by confirmation from user.